client/src/pages/login/index.js

```diff
@@ -1,33 +1,88 @@
 import React, { Component } from "react";
+import SweetAlert from "react-bootstrap-sweetalert";
```

Comment:
(I'm copying this comment over from #26)

Cool! I've never seen SweetAlert before.
That makes me think of Glyphicons, maybe they'll be useful to you at some point:
https://react-bootstrap.github.io/components/glyphicons/
http://glyphicons.com/